content/pages/disability-benefits/conditions.md

```diff
@@ -3,6 +3,7 @@ layout: page-breadcrumbs.html
 title: Conditions
 source: https://eauth.va.gov/ebenefits/learn/compensation
 template: 1-topic-landing
+lastupdate: 11-24-2016
```

Comment:
What format is this date supposed to be in? My assumption is that it should be YYYY-MM-DD, but maybe not.